angular/src/components/set-password.component.ts

```diff
@@ -69,30 +82,51 @@ export class SetPasswordComponent extends BaseChangePasswordComponent {
 
     async performSubmitActions(masterPasswordHash: string, key: SymmetricCryptoKey,
         encKey: [SymmetricCryptoKey, EncString]) {
+        // Create Set Password Request
         const request = new SetPasswordRequest();
         request.masterPasswordHash = masterPasswordHash;
         request.key = encKey[1].encryptedString;
         request.masterPasswordHint = this.hint;
         request.kdf = this.kdf;
         request.kdfIterations = this.kdfIterations;
         request.orgIdentifier = this.identifier;
-
+        // Create Key Pair
```

Comment:
Same, this comment's existence is indicative of an issue with the function. I would include keys in the above mentioned request constructor.

Reply:
Done!